silnlp/common/environment.py

```diff
@@ -12,6 +12,9 @@ def get_data_dir() -> Path:
     sil_nlp_data_path = os.getenv("SIL_NLP_DATA_PATH")
     if sil_nlp_data_path is not None:
         return Path(sil_nlp_data_path)
+    auqa_ml_path = Path("/data/aqua-ml-data")
```

Comment:
rclone should be mounted to the path specified by `SIL_NLP_DATA_PATH`, then this code won't be necessary.